docs/source/examples/demo_transformer_model_train_save_upload_to_openSearch.ipynb

```diff
@@ -1,5 +1,22 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "a956cfe2",
+   "metadata": {},
+   "source": [
+    "# Demo Notebook for Sentence Transformer Model Training, Saving and Uploading to OpenSearch\n",
+    "This notebook provides a walkthrough guidance for users use their synthetic queries to fine tune and train a sentence transformer model. In this notebook, you use opensearch_py_ml to accomplish the following:\n",
+    "\n",
+    "Step 0: Import packages and set up client\n",
+    "\n",
+    "Step 1: Read synthetic queries and train/fine-tune model using a hugging face sentence transformer model\n",
```

Comment:
Will we create a new demo notebook to show how to generate synthetic queries?

Reply:
How about we also create a new demo notebook about how to trace huggingface model to torchscript and upload to ml-commons?

Reply:
Will we create a new demo notebook to show how to generate synthetic queries?

--> Yes we will have another demo notebook to show how to generate synthetic queries. 

How about we also create a new demo notebook about how to trace huggingface model to torchscript and upload to ml-commons?

--> That's a good idea. We can add that too.

Reply:
I will add these two notebooks in a separate PR.